main.py

```diff
@@ -535,6 +535,9 @@ def runPeriodically(self, *args):
                     print message
                     measuredDist = float(message[9:len(message)-3])
                     self.data.measureRequest(measuredDist)
+            elif message[0:13] == "Maslow Paused":
```

Comment:
Is there a cleaner way to do this than looking at a range of characters? I know this is my fault (I started it), but I feel like maybe we should be doing a search to see if the string "Maslow paused" appears in the command or something

Reply:
I was just following the example from the other parts of the if statement.  These will be the first characters in the message, so I think we're okay.  If you want to experiment with a string search, that's fine too!